packages/components/src/util/typography.tsx

```diff
@@ -93,6 +100,11 @@ function Typography<IComponent extends React.ElementType>({
         // Weights
         weight === "bold" && styles.weightBold,
         weight === "normal" && styles.weightNormal,
+        // Spacing
+        spacing === "none" && styles.spacingNone,
```

Comment:
@anniehu4 Looking for recommendations on whether this is a good starting point for spacing or if there's a better resource for defaults.

Reply:
I'd def recommend including `mb-1`, and I'm not suuuper sure there's a need for `mb-8` 🤔  given that, maybe we should also think of spacing in multiples of 8px

reasoning is most of traject's headings currently have `5px` margin bottom ([claro/_components/_app.scss](https://github.com/FB-PLP/traject/blob/cefa996b6c9593581f76364c4c6ad36fc20f4c8e/app/assets/stylesheets/claro/_components/_app.scss#L29)) and of the spacing variables in `_variables.scss`:
```
// Spacing
$spacing-none: 0;
$spacing-xxs: 2px;
$spacing-xs: 5px;
$spacing-sm: 10px;
$spacing-md: 15px;
$spacing-lg: 20px;
$spacing-lg-2: 25px;
$spacing-xl: 30px;
```

I only commonly see up to 20px being used

Reply:
Oh no - I'm pretty sure our design docs mention our spacing as multiples of 8px, but maybe that's too optimistic 🤔

Reply:
you know... it actually might be fine 🤔  maybe everything is too squished rn and 8px would actually look nicer

Reply:
actually, you know what -- down to try it and revisit if we do need smaller 🙂  I trust sean's eye more than mine